src/flow.h

```diff
@@ -312,7 +312,8 @@ typedef struct FlowKey_
     Port sp, dp;
     uint8_t proto;
     uint8_t recursion_level;
-    uint16_t vlan_id[2];
+    struct LiveDevice_ *livedev;
```

Comment:
why do we store a pointer here instead of the id?

Also in IPS mode there are generally 2, one per direction. How will that be handled?

Reply:
@catenacyber Phillipe, can you address this comment?

Reply:
Done in https://github.com/OISF/suricata/pull/7982